changelogs/unreleased/4325-delete-service-button.yml

```diff
@@ -0,0 +1,8 @@
+change-type: patch
+description: 'Move delete service functionality to catalog view'
```

Comment:
Move delete service button to the Service Catalog overview, in the kebab menu options.